nixos/tests/hydra/default.nix

```diff
@@ -7,9 +7,8 @@ let
      { trivial = builtins.derivation {
          name = "trivial";
          system = "x86_64-linux";
-         PATH = coreutils;
```

Comment:
The `with import <nix/config.nix>;` line is then useless. Could you also remove it?